### DISCUSSION TITLE
i can not sent xch from my wallet to exchange

### DISCUSSION THREAD

**Discussion by chelsealoveu** [Chia-Network/chia-blockchain#17128] [Support]
I can't transfer XCH from wallet to exchange? pending status? Is there anyone like me?

**Comment by zhang808908** [comment]
我也提不出来，我设置了0.00001的手续费也不行

**Comment by BrandtH22** [comment]
Hey @chelsealoveu and @zhang808908 , please note that we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

Over the past couple of weeks we have been experiencing a very active chain with most blocks being mostly full. This can also be considered fee pressure and means that to transact on chia one needs to set a reasonable fee based on the transaction type they are sending. 

We have created a reference doc of estimated fees but please note that the fees listed in this document represent the minimums for the fee to take effect so based on the current fee pressure you may need to include a higher fee to have a time-sensitive transaction processed in a timely manner): https://docs.chia.net/coin-set-costs/#estimated-transaction-costs

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
